app/src/main/java/com/android/halanvendordiscovery/domain/details/model/MerchantDomainModel.kt

```diff
@@ -0,0 +1,12 @@
+package com.android.halanvendordiscovery.domain.details.model
+
+data class MerchantDomainModel(
+    val name            : String?,
+    val arabicName      : String?,
+    val address         : String?,
+    val longitude       : Double?,
+    val latitude        : Double?,
+    val phoneNumber     : String?,
+    val vendorName      : String?,
+    val vendorArabicName: String?
+)
```

Comment:
is this model response from backend is all nullable ?

Reply:
This model is updated in the domain layer that has been merged with feature branch